neqo-transport/src/tracking.rs

```diff
@@ -351,6 +353,23 @@ pub struct AckToken {
     ranges: Vec<PacketRange>,
 }
 
+/// The counts for different ECN marks.
+pub type EcnCount = EnumMap<IpTosEcn, u64>;
+
+/// Whether any of the ECN counts are non-zero.
+pub fn have_ecn_count(ecn: &EcnCount) -> bool {
+    ecn[IpTosEcn::Ect0] > 0 || ecn[IpTosEcn::Ect1] > 0 || ecn[IpTosEcn::Ce] > 0
+}
+
+/// Subtract the ECN counts in `b` from `a`.
+pub fn diff_ecn_count(a: &EcnCount, b: &EcnCount) -> EcnCount {
```

Comment:
Given that you have two utility functions attached to this, I would change from a type alias to a simple struct wrapper:

```rust
struct EcnCount(EnumMap<IpTosEcn, u64>);
```

Then, I would recommend implementing `std::ops::Sub` for this.  It's saturating, but it's still a fine use of the feature.

Are we counting non-ect?  If not, this might be a more complicated structure, but a leaner one as a result.

Reply:
We are currently counting Not-ECT.